reactor-core/src/test/java/reactor/core/publisher/FluxFlattenIterableTest.java

```diff
@@ -398,7 +397,7 @@ public void syncDrainWithPollFailure() {
     }
 
 	@Test
-	public void errorModeContinueNullPublisher() {
+	public void errorModeContinueNullPublisherAsync() {
```

Comment:
nit: maybe replace the test name suffixes of `Async` with `NotFused` and `Sync` with `Fused` ?

Reply:
Sure. I thought the Async / sync was logical because of the drain method that would get called.
Will change.